podpac/core/node.py

```diff
@@ -95,17 +104,39 @@ class Node(tl.HasTraits):
 
     units = Units(default_value=None, allow_none=True)
     dtype = tl.Any(default_value=float)
+    cache_output = tl.Bool(True)
+    cache_update = tl.Bool(False)
     cache_type = tl.Enum([None, 'disk', 'ram'], allow_none=True)
+    cache_ctrl = tl.Instance(cache.CacheCtrl, allow_none=True)
+
+    @tl.default('cache_ctrl')
+    def _cache_ctrl_default(self):
```

Comment:
The default should really probably be a ram store. I'm not sure what to do about SMAP for example...

Reply:
When we add RAM cache we'll probably need to make the default cache_ctrl a global variable (or at least the RAM store) so that shared data structures can be used.